ddl/src/main/java/com/adaptivescale/rosetta/ddl/targets/mysql/MySqlDDLGenerator.java

```diff
@@ -50,6 +50,10 @@ public String createTable(Table table, boolean dropTableIfExists) {
             stringBuilder.append("`").append(table.getName()).append("`").append("; \n");
         }
 
+        if (table.getSchema() != null && !table.getSchema().isBlank()) {
```

Comment:
same issue as in the above comment - i think it might be a better check do this on db level - where we check whether we need to create schema(s) in some kind of db level schema list and inject them into query after we generate all the table checks (i.e. list of schemas to be created)